### ISSUE TITLE
Intelli Sense Error Squiggles

### ISSUE THREAD

**Issue by Botan626** [L-I-V/MQL-Tools#53]
Hi,
E.g. I have 1 error.
![image](https://github.com/L-I-V/MQL-Tools/assets/28785698/7ab993ea-0025-451e-a496-6b177f251cdf)

Problems:
1. It shows wrong file name, the real one is "HA+MM+PP.mq5".
2. If I click on the error, vscode will try to open that non-existent file with this error:
![image](https://github.com/L-I-V/MQL-Tools/assets/28785698/8d7da35d-5056-455c-bac3-b242b25330f5)

Compile with script works fine.

**Comment by pungggi** [comment]
Do you have in your settings.json the following?: `"C_Cpp.errorSquiggles": "disabled",`

**Comment by Botan626** [comment]
> Do you have in your settings.json the following?: `"C_Cpp.errorSquiggles": "disabled",`

No, but I have `"mql_tools.IntelliSenseErrorSquiggles": false`

**Comment by aipungggi** [comment]
and did you try "C_Cpp.errorSquiggles": "disabled" ?

**Comment by Botan626** [comment]
> and did you try "C_Cpp.errorSquiggles": "disabled" ?

No, but now I turned Intelli Sense on and this problem is gone.

**Comment by Botan626** [comment]
> and did you try "C_Cpp.errorSquiggles": "disabled" ?

Now I tried, and it didn't help, and it seems like file dependent, for one file clicking on the problem opens corresponding line, but for the file from 1st comment behaviour is the same.

**Comment by Botan626** [comment]
it's not file name dependent

same issue with `Double smoothed EMA (mtf).mq5` indicator

adding `_@!!@` to the file name and trying to open it

![image](https://github.com/L-I-V/MQL-Tools/assets/28785698/8728daa6-bf4b-4b3a-8d26-129066a4584e)
